contracts/spore/build.rs

```diff
@@ -28,14 +28,17 @@ pub fn main() {
     let cluster_code_hash = load_code_hash("cluster", &compile_mode);
     let cluster_agent_code_hash = load_code_hash("cluster_agent", &compile_mode);
 
-    let mut cluster_code_hashes = vec![cluster_code_hash];
-    // this is version v1 of cluster contract in testnet
-    cluster_code_hashes.push(
-        hex::decode("598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080")
-            .unwrap()
-            .try_into()
-            .unwrap(),
-    );
+    let mut cluster_code_hashes = vec![];
+    if let Err(env::VarError::NotPresent) = env::var("CARGO_FEATURE_RELEASE_EXPORT") {
+        // this is version v1 of cluster contract in testnet
+        cluster_code_hashes.push(
+            hex::decode("598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080")
```

Comment:
- Which commit generated this cluster_code_hash(`598d793defef36e2eeba54a9b45130e4ca92822e1d193671f490950c3b856080`)?

- What about the latest data_hash: `0x15f835c4ca0b861df38f10d4e95c51ba9cee3c89f178b21e2e28baa67ebd8b42` recorded in README?

Reply:
because this new one is provided by ShiSong, and this version contains bugs which I've been solving after my take over, so the old one (previous version v1) is the stable one and should be injected